pyam/kaya/lmdi_names.py

```diff
@@ -0,0 +1,7 @@
+Pop_LMDI = "Population (LMDI)"
```

Comment:
Following the current IAMC-variable naming convention, it would be preferably to use square brackets for these identifiers.
```suggestion
Pop_LMDI = "Population [LMDI]"
```

Reply:
will change in separate pull request for lmdi feature